### PR TITLE
Adding linked data links to swissboundaries

### PR DESCRIPTION
**Pull request by gjn**
This is a fix for https://github.com/geoadmin/mf-geoadmin3/issues/3899

[Testlink](https://mf-geoadmin3.int.bgdi.ch/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fgjn_ld&lang=en&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.swissboundaries3d-gemeinde-flaeche.fill,ch.swisstopo.swissboundaries3d-bezirk-flaeche.fill,ch.swisstopo.swissboundaries3d-kanton-flaeche.fill)

Ping @p1d1d1 @davidoesch

### CODE REVIEW

**Comment by p1d1d1** [comment]
@gjn please open link in a new tab

**Comment by davidoesch** [comment]
New Tab- otherwise +1

**Comment by gjn** [comment]
Adapted.

**Comment by p1d1d1** [comment]
+1 for me

File: chsdi/templates/htmlpopup/swissboundaries_bezirk.mako [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c,lang)">
+<%
+linkeddatahost = request.registry.settings['linkeddatahost']
+ldlink = linkeddatahost + '/boundaries/district/' + str(c['featureId'])
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/swissboundaries_bezirk.mako]: if I may suggest, featureId is mainly used for backward compatibility issues, I would like to get rid of these duplicated ids at some point, so always prefer id.

File: rc_dev [hunk 2]
```diff
@@ -21,3 +21,4 @@ export VECTOR_PROFILENAME=chsdi_aws_admin_ro
 export WMSHOST=wms-bgdi.dev.bgdi.ch
 export WSGI_THREADS=15
 export CMSGEOADMINHOST=https://cms.geo.admin.ch
+export LINKEDDATA_HOST=https://ld.geo.admin.ch
```
Comment (loicgasser) [inline rc_dev]: All hosts don't have underscore, or at least keep the convention env variable with caps and app variable without,

File: chsdi/templates/htmlpopup/swissboundaries_bezirk.mako (hunk 1 above)
Comment (gjn) [inline chsdi/templates/htmlpopup/swissboundaries_bezirk.mako]: done.

File: rc_dev (hunk 2 above)
Comment (gjn) [inline rc_dev]: I would prefer with underscore, it's more readable.

I've adapted because of consistency.

**Comment by loicgasser** [comment]
yes underscore is better, can be done as part of https://github.com/geoadmin/mf-chsdi3/pull/2619

**Pull request merged**